assignment-sweet-compose/answer/docker-compose.yml

```diff
@@ -35,17 +34,17 @@ services:
 
   db:
     image: postgres:9.6
-    environment:
```

Comment:
It seems this line caused the error message `FATAL:  role "root" does not exist`. And there is no such a string in the video

Reply:
A few months back I updated the code to auth to postgres, and forgot to change this line, thanks, I'll fix.